src/reducers/index.js

```diff
@@ -1,5 +1,94 @@
-export default (state = 0, action) => {
+export default (state = {}, action) => {
   switch (action.type) {
-   //actions go here
+    case 'LOAD':
+      return state = {
+        "Type": "MyObjectBuilder_Refinery/LargeRefinery",
+        "Name": "Refinery",
+        "Id": 110679305426212711,
+        "Inventories": [
+          {
+            "CurrentVolume": 0.0,
+            "MaxVolume": 9.223372E+12
+          },
+          {
+            "CurrentVolume": 0.013916,
+            "MaxVolume": 9.223372E+12,
+            "Items": [
+              {
+                "Type": "MyObjectBuilder_Ingot/Cobalt",
+                "Amount": 33.9410858,
+                "Mass": 33.9410858,
+                "Volume": 0.0038014017
+              },
+              {
+                "Type": "MyObjectBuilder_Ingot/Gold",
+                "Amount": 1.131289,
+                "Mass": 1.131289,
+                "Volume": 5.88270268E-05
+              },
+              {
+                "Type": "MyObjectBuilder_Ingot/Iron",
+                "Amount": 79.19589,
+                "Mass": 79.19589,
+                "Volume": 0.0100578787
+              }
+            ]
+          }
+        ],
+        "Properties": [
+          {
+            "Name": "OnOff",
+            "Type": "Boolean",
+            "Value": true
+          },
+          {
+            "Name": "ShowInTerminal",
+            "Type": "Boolean",
+            "Value": true
+          },
+          {
+            "Name": "ShowInInventory",
+            "Type": "Boolean",
+            "Value": true
+          },
+          {
+            "Name": "ShowInToolbarConfig",
+            "Type": "Boolean",
+            "Value": true
+          },
+          {
+            "Name": "Name",
```

Comment:
On my instance of this branch -- I've got a blank between show in toolbar config and show on hud.  It looks like that checkbox is drawn because of the stringbuilder state.Properties.Name property (which happens to be the same as state.Name above).  

Is it easy-ish for components/Checkbox.js to only attempt to render properties that are of type "Boolean" ? 

<img width="207" alt="screen shot 2017-02-01 at 8 31 16 pm" src="https://cloud.githubusercontent.com/assets/2091058/22537216/6b8d05b4-e8bd-11e6-9e2e-2012a9b3db23.png">